gradle.properties

```diff
@@ -3,6 +3,8 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+systemProp.bwc.version=2.12.0-SNAPSHOT
```

Comment:
We should keep latest version here because in wire dependency between older versions 3.x is compatible with 2.(latest-major)(i.e. 2.12-SNAPSHOT). Let say if we set 2.9 and If someone clones the project just hit ./gradlew :qa:bwcTestSuite then rolling upgrade tests will fail. He has to manually update the version here with the latest compatible version. This has been kept just to run tests easily on local machine with zero effort.

Reply:
Still if you suggest to keep it 2.9 then I will update it.

Reply:
That makes sense, I think we're good with 2.12 then